### PR TITLE
[UI] Initial use of Sistent components

### PR DESCRIPTION
**Pull request by sudhanshutech**
**Notes for Reviewers**

This PR add the use sistent components from layer5/sistent

>[!NOTE]
>  1. Only available components are added in such way nothing breaks
>  2. Currently some components are not available in sistent so if a unavailable and its wrapped under a available component then some of its feature is not working including styles so excluding them initially
> 3. As new components will be added and released soon here will be updated

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by leecalcote** [comment]
@sudhanshutech @nebula-aac --> https://github.com/meshery/meshery/actions/runs/6526099532/job/17719484339?pr=9122

**Comment by sudhanshutech** [comment]
> @sudhanshutech @nebula-aac --> https://github.com/meshery/meshery/actions/runs/6526099532/job/17719484339?pr=9122

@nebula-aac i think it is from sistent. Since building is failing there also right?

**Comment by sudhanshutech** [comment]
@nebula-aac still failing

File: ui/components/User.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import { List, ListItem } from '@material-ui/core';
-import Avatar from '@material-ui/core/Avatar';
+import { Avatar } from '@layer5/sistent-components';
```
Comment (Ckbhatia) [inline ui/components/User.js:2]: Does this support tree-shaking?

**Comment by leecalcote** [comment]
Merge conflict, @sudhanshutech

**Comment by leecalcote** [comment]
Merge conflict, @sudhanshutech

**Pull request closed without merging**